src/BulletDynamics/Featherstone/btMultiBodyDynamicsWorld.cpp

```diff
@@ -428,13 +428,18 @@ void btMultiBodyDynamicsWorld::forwardKinematics()
 void btMultiBodyDynamicsWorld::solveConstraints(btContactSolverInfo& solverInfo)
 {
     solveExternalForces(solverInfo);
-    buildIslands();
+    m_islandManager->buildAndProcessIslands(getCollisionWorld()->getDispatcher(), getCollisionWorld(), m_solverMultiBodyIslandCallback);
```

Comment:
best to undo island changes, and derive a class and assign m_solverMultiBodyIslandCallback in btDeformableMultiBody...World.cpp constructor. Make solveConstraint virtual, and override with CG solver/new logic.